### PR TITLE
Update the README and add buttons

### PR DESCRIPTION
**Pull request by Poussinou**
Add two buttons with links to fdroid and google play store

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 The Android port of the Frozen Bubble game.  Originally developed with SDK
 v21.1.0, using API level 4 (version 1.6 - Donut).
 
+<a href="https://f-droid.org/repository/browse/?fdid=org.jfedor.frozenbubble" target="_blank">
+<img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="80"/></a>
+<a href="https://play.google.com/store/apps/details?id=org.jfedor.frozenbubble&hl=fr" target="_blank">
```
Comment (sm4rk0) [inline README.md]: `&hl=fr` part of the Play Store URL forces the page to load in French.

Comment (Poussinou) [inline README.md]: sorry about that, I forgot to remove it

Comment (sm4rk0) [inline README.md]: No problem. Merci for contributing! (BTW, I'm just a contributor like you)

**Comment by videogameboy76** [comment]
Thank you gentlemen, very nice.

**Pull request merged**